CalibMuon/DTCalibration/python/dtCalibOfflineSelection_cff.py

```diff
@@ -16,8 +16,9 @@
 hltDTCalibration.HLTPaths = ['HLT_DTCalibration_v*']
 
 ALCARECODtCalibHIHLTFilter = copy.deepcopy(hltHighLevel)
-ALCARECODtCalibHIHLTFilter.throw = False
-ALCARECODtCalibHIHLTFilter.eventSetupPathsKey = 'MuAlcaDtCalibHI'
+
+ALCARECODtCalibHIHLTFilter.HLTPaths = ['HLT_HIL2SingleMu*']
+
```

Comment:
```suggestion
ALCARECODtCalibHIHLTFilter.HLTPaths = ['HLT_HIL2SingleMu*']
```
(not worth updating the PR only for this, but if you need to for a different reason please take it into account)